src/pages/Profile.tsx

```diff
@@ -21,14 +21,15 @@ export function Profile() {
       }}
     >
       <Box sx={{ position: "relative", mb: 3 }}>
-        <Avatar
-          src={avatar}
-          alt="Your avatar"
-          sx={{
-            height: 112,
-            width: 112,
-          }}
-        />
+        {/* To be brought back in the next iteration */}
+        {/*<Avatar*/}
+        {/*  src={avatar}*/}
+        {/*  alt="Your avatar"*/}
+        {/*  sx={{*/}
+        {/*    height: 112,*/}
+        {/*    width: 112,*/}
+        {/*  }}*/}
+        {/*/>*/}
```

Comment:
The current user model on Supabase doesn't yet support avatars, so this code is commented out. I plan to add avatar support in the next iteration and will re-enable this functionality then